ftw/workspace/zip_export.py

```diff
@@ -0,0 +1,23 @@
+from ftw.pdfgenerator.interfaces import IPDFAssembler
+from ftw.workspace.interfaces import IWorkspace
+from ftw.zipexport.interfaces import IZipRepresentation
+from ftw.zipexport.representations.general import NullZipRepresentation
+from StringIO import StringIO
+from zope.component import adapts
+from zope.component import getMultiAdapter
+from zope.interface import implements
+from zope.interface import Interface
+
+
+class WorkspaceZipRepresentation(NullZipRepresentation):
```

Comment:
This breaks exporting subobjects.
Can you make this subclass ``FolderZipRepresentation`` and also yield the files of the superclass ``get_files`` in our ``get_files``?